lib/run/summary.js

```diff
@@ -2,7 +2,11 @@ var _ = require('lodash'),
     SerialiseError = require('serialised-error'),
 
     proxyPropertyList,
-    RunSummary;
+    RunSummary,
+    exportReference = {
```

Comment:
That is the correct way of doing things, but is not possible right now as globals and environments are handled via differing formats.